SU2_CFD/src/iteration/CTurboIteration.cpp

```diff
@@ -64,3 +64,78 @@ void CTurboIteration::InitTurboPerformance(CGeometry* geometry, CConfig** config
   TurbomachineryPerformance = std::make_shared<CTurboOutput>(config, *geometry, *fluid);
   TurbomachineryStagePerformance = std::make_shared<CTurbomachineryStagePerformance>(*fluid);
 }
+
+void CTurboIteration::TurboRamp(CGeometry**** geometry_container, CConfig** config_container, unsigned long iter, unsigned short iZone, TURBO_RAMP_TYPE ramp_flag) {
+  /*--- Generic function for handling turbomachinery ramps ---*/
+  // Grid updates (i.e. rotation/translation) handled seperately to boundary (i.e. pressure/mass flow) updates
+  auto* config = config_container[iZone];
+  auto* geometry = geometry_container[iZone][INST_0][ZONE_0];
+
+  std::string msg = "\nUpdated rotating frame grid velocities for zone ";
+  if (ramp_flag == TURBO_RAMP_TYPE::GRID && config->GetKind_GridMovement() == ENUM_GRIDMOVEMENT::STEADY_TRANSLATION) 
+    msg = "\nUpdated translation frame grid velocities for zone ";
+
+  if (config_container[ZONE_0]->GetMultizone_Problem())
+    iter = config_container[ZONE_0]->GetOuterIter();
+
+  /*-- Update grid velocities (ROTATING_FRAME, STEADY_TRANSLATION)*/
+  if (ramp_flag == TURBO_RAMP_TYPE::GRID && config->GetGrid_Movement()) {
+    const auto ini_vel = config->GetRamp_Coeff(ramp_flag, TURBO_RAMP_COEFF::INITIAL_VALUE);
+    const long unsigned rampFreq = SU2_TYPE::Int(config->GetRamp_Coeff(ramp_flag, TURBO_RAMP_COEFF::UPDATE_FREQ));
+    const long unsigned finalRamp_Iter = SU2_TYPE::Int(config->GetRamp_Coeff(ramp_flag, TURBO_RAMP_COEFF::FINAL_ITER));
```

Comment:
Any reason why you can't auto these?
```suggestion
    const unsigned long rampFreq = SU2_TYPE::Int(config->GetRamp_Coeff(ramp_flag, TURBO_RAMP_COEFF::UPDATE_FREQ));
    const unsigned long finalRamp_Iter = SU2_TYPE::Int(config->GetRamp_Coeff(ramp_flag, TURBO_RAMP_COEFF::FINAL_ITER));
```

Reply:
they compare with the value from config->GetOuterIter, which is a long unsigned int and returns an error in the compilation that fails the reg tests.